src/llscan.cc

```diff
@@ -1141,6 +1163,56 @@ uint64_t FindJSObjectsVisitor::Visit(uint64_t location, uint64_t word) {
 }
 
 
+void FindJSObjectsVisitor::InsertOnMapsToInstances(
+    uint64_t word, v8::Map map, FindJSObjectsVisitor::MapCacheEntry map_info,
+    v8::Error& err) {
+  TypeRecord* t;
+
+  /* No entry in the map, create a new one. */
+  if (llscan_->GetMapsToInstances().count(map_info.type_name) == 0) {
+    t = new TypeRecord(map_info.type_name);
+    llscan_->GetMapsToInstances().emplace(map_info.type_name, t);
+  } else {
+    /* Get entry from the map. */
+    t = llscan_->GetMapsToInstances().at(map_info.type_name);
+  }
```

Comment:
I realize this comes from existing code but `.count()` is an O(n) function.  `.find()` would be better but since you're inserting, this is even more concise and efficient:
```c++
auto entry = std::make_pair(map_info.type_name, nullptr);
auto pp = &llscan_->GetMapsToInstances().insert(entry).first->second;
if (*pp == nullptr) *pp = new TypeRecord(map_info.type_name);
t = *pp;
```